components/empty-state/test/empty-state-illustrated-button.test.js

```diff
@@ -0,0 +1,27 @@
+import '../empty-state-illustrated-button.js';
+import { fixture, html, oneEvent } from '@open-wc/testing';
+import { runConstructor } from '../../../tools/constructor-test-helper.js';
+
+describe('d2l-empty-state-illustrated-button',  () => {
```

Comment:
My concern with this is that it could slow down our unit tests a lot. After a discussion with Dave L that attributes may not be set instantly, I am now checking for missing attributes after a 3 second delay. Checking all the error messages for my components would add an additional 18+ seconds.

Reply:
Ah yup that sounds like a valid concern